packages/webapp/src/components/Table/types.ts

```diff
@@ -0,0 +1,82 @@
+/*
+ *  Copyright 2024 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+import type { ReactElement, ReactNode } from 'react';
+import type { ColumnInstance } from 'react-table';
+import { ReactComponentLike } from 'prop-types';
+
+export enum TableKind {
+  V1 = 'v1',
+  V2 = 'v2',
+}
+
+export enum CellKind {
+  HOVER_PILL_OVERFLOW = 'hoverPillOverflow',
+  ICON_TEXT = 'iconText',
+  PLAIN = 'plain',
+  RIGHT_CHEVRON_LINK = 'rightChevronLink',
+}
+
+enum Alignment {
+  LEFT = 'left',
+  RIGHT = 'right',
+}
+
+type CallbackFn = () => void;
+
+export type ReactTableProps = {
```

Comment:
I think they belong in their respective tables files since they are equivalent of "PropTypes" (not prototype haha ), ...but those files are in .jsx right now and I had to stop somewhere. 

So my preference after that was a shared component `types.ts` file instead of "some other file that primarily uses it".